### PR TITLE
{data,lang,lib,vis}[GCCcore/13.2.0] FFmpeg v6.0, LAME v3.100, FriBidi v1.0.13, ...

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Micket** [comment]
@boegelbot please test @ generoso

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
alvis-c1 - Linux Rocky Linux 8.8, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/Micket/08cdf95ea9080798fc8151c20ae38b43 for a full test report.

File: easybuild/easyconfigs/f/FriBidi/FriBidi-1.0.13-GCCcore-13.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+checksums = ['7fa16c80c81bd622f7b198d31356da139cc318a63fc7761217af4130903f54a2']
+
+builddependencies = [
+    ('binutils', '2.40'),
+    ('pkgconf', '2.0.3'),
+    ('Autotools', '20220317'),
+]
+
+configopts = '--disable-docs'
+
```
Comment (branfosj) [inline easybuild/easyconfigs/f/FriBidi/FriBidi-1.0.13-GCCcore-13.2.0.eb:25]:
```suggestion
```

Remove (see #19147)

**Comment by Micket** [comment]
@boegelbot please test @ jsc-zen2

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in total)
alvis-c1 - Linux Rocky Linux 8.8, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/Micket/cdeccc08fe4da2f8025c69a05a61f194 for a full test report.

File: easybuild/easyconfigs/f/ffnvcodec/ffnvcodec-12.1.14.0.eb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+sources = [{
+    'git_config': {
+        'url': 'https://git.videolan.org/git/ffmpeg/',
+        'repo_name': 'nv-codec-headers',
+        'tag': 'n%(version)s',
+    },
+    'filename': SOURCE_TAR_GZ,
+}]
+checksums = ['a7da1532e61dee98c2a917a7a3f9127b94a1d72d44aa1cce23e3d797b114d0c0']
```
Comment (branfosj) [inline easybuild/easyconfigs/f/ffnvcodec/ffnvcodec-12.1.14.0.eb:21]:
```suggestion
checksums = [None]
```

As it is a git download.

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
@boegelbot please test @ jsc-zen2

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 8 out of 8 (5 easyconfigs in total)
bear-pg0105u03a - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/8681d853229f0de743c292007910394e for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in total)
zen2-ubuntu-2310 - Linux Ubuntu 23.10 (Mantic Minotaur), x86_64, AMD EPYC 7452 32-Core Processor (zen2), Python 3.11.6
See https://gist.github.com/SebastianAchilles/ac290de819cfa714a0869be611fba310 for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @Micket!

**Pull request merged**